### PR TITLE
update changes.

### PR DESCRIPTION
**Pull request by Palak4620**

### CODE REVIEW

File: oshi-core/src/main/java/oshi/annotation/concurrent/SuppressForbidden.java
Comment (dbwiddis) [inline oshi-core/src/main/java/oshi/annotation/concurrent/SuppressForbidden.java:1]: 1. Just put this in `oshi.annotation`, not the concurrent subpackage.
2. You'll need to update the import statements where this is used in `oshi.demo`.

File: pom.xml [hunk 1]
```diff
@@ -161,6 +161,25 @@
...
+                    </manifest>
+                    <manifestEntries>
+                        <!-- Add other manifest entries if needed -->
+                    </manifestEntries>
+                </archive>
+                <signaturesFiles>
+                    <signaturesFile><your-signatures-file></signaturesFile>
+                </signaturesFiles>
+            </configuration>
+        </plugin>
```
Comment (dbwiddis) [inline pom.xml:182]: This is the wrong plugin to add.  Go to the `forbiddenapis` plugin here:
https://github.com/oshi/oshi/blob/2a37177fe9c9c5c96f6711b62d7b8c97814f0d09/pom.xml#L746-L770

You'll be adding the `<signatureFiles>` section inside configuration.  See sample POM entry [here](https://github.com/policeman-tools/forbidden-apis/wiki/MavenUsage#maven-usage-instructions).  Format of that file (you can put it in the `config` directory under root) is [here](https://github.com/policeman-tools/forbidden-apis/wiki/SignaturesSyntax#syntax-of-forbidden-signatures).

**Review by dbwiddis — CHANGES_REQUESTED** [review]

File: oshi-core/src/main/java/oshi/annotation/concurrent/SuppressForbidden.java
Comment (Palak4620) [reply oshi-core/src/main/java/oshi/annotation/concurrent/SuppressForbidden.java:1]: okay, I will do that.

**Comment by dbwiddis** [comment]
Superceced by #2474

**Pull request closed without merging**